def raise_error(type: str, message: str):
    print(f"[{type}] Error: {message}")
    exit(1)